docker/pytorch/Dockerfile

```diff
@@ -1,18 +1,18 @@
-########## PyTorch 1.4 Base #################
+########## PyTorch 1.5 Base #################
```

Comment:
Let's remove the minor version numbers from these and just have PyTorch 1 Base, TF 2 Base, etc.